src/sky.cpp

```diff
@@ -202,15 +202,14 @@ void Sky::render()
 
 		driver->setMaterial(m_materials[1]);
 
-		video::SColor cloudyfogcolor = m_bgcolor;
-
-		// Draw far cloudy fog thing blended with skycolor
+		// Draw top band of our smooth skybox
 		for (u32 j = 0; j < 4; j++) {
-			video::SColor c = cloudyfogcolor.getInterpolated(m_skycolor, 0.45);
-			vertices[0] = video::S3DVertex(-1, 0.08, -1, 0, 0, 1, c, t, t);
-			vertices[1] = video::S3DVertex( 1, 0.08, -1, 0, 0, 1, c, o, t);
-			vertices[2] = video::S3DVertex( 1, 0.12, -1, 0, 0, 1, c, o, o);
-			vertices[3] = video::S3DVertex(-1, 0.12, -1, 0, 0, 1, c, t, o);
+			video::SColor c = m_bgcolor;
+			video::SColor c_blend = m_skycolor; //
+				vertices[0] = video::S3DVertex(-1, -0.1, -1, 0, 0, 1, c, t, t);
+				vertices[1] = video::S3DVertex( 1, -0.1, -1, 0, 0, 1, c, o, t);
+				vertices[2] = video::S3DVertex( 1, 0.3, -1, 0, 0, 1, c_blend, o, o);
+				vertices[3] = video::S3DVertex(-1, 0.3, -1, 0, 0, 1, c_blend, t, o);
```

Comment:
Any higher and you start getting colour-banding due to limited RGB space. It'd still be effectively invisible due to the lack of contrast of the default sky colours.

Reply:
I don't mean the bgcolor/skycolor blend should reach higher, i mean the opposite.
For a blue gradient reaching from horizon to 1.0, the colours could be tuned to not create visual banding.

Reply:
Banding can always add some charm to a boring bland skybox, almost as if it's a game using dithering. I obviously want to avoid it, but can be a charmer, sort of.